tests/parsers/pomparser/resources/forbidden_entity.xml

```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
+<!DOCTYPE xml [<!ENTITY quot "&#34;">]>
```

Comment:
Just curious, does `defusedxml` ban all usages of `ENTITY` in any XML document provided to it, or does it only ban case by case?
I am a bit unclear which scenario this test case falls into for `defusedxml` to trigger an exception.